docs/docs/guides/basics/eth.md

```diff
@@ -344,6 +343,838 @@ Transaction receipt: {
+
+    console.log('Current balance:', currentBalance); // 100000000000000000000n
+}
+(async () => {
+    await test();
+})();
+```
+
+### getBlock
+
+The [getBlock](/api/web3-eth/function/getBlock) function is used to retrieve information about a specific Ethereum block. Ethereum blocks are the fundamental building blocks of the blockchain, containing a collection of transactions and other data.
+
+The [getBlock](/api/web3-eth/function/getBlock) function typically takes two parameters:
+- block ([HexString32Bytes](/api/web3/namespace/types/#HexString32Bytes) | [BlockNumberOrTag](/api/web3/namespace/types/#BlockNumberOrTag)) - Number of block or tag
+- hydrated (boolean) -  if true, it returns the detail of each transaction. If false, only the hashes of the transactions.
+
+The function returns a Promise that resolves to an object containing details about the requested block. The structure of this object may vary depending on whether you requested transaction objects. The block information often includes fields like number, hash, parentHash, nonce, miner, gasLimit, gasUsed, timestamp, and more.
+```typescript
+import Web3 from 'web3';
+const web3 = new Web3('http://localhost:7545');
+
+async function test() {
+    const block = await web3.eth.getBlock('latest');
+    console.log('Block:', block);
+    // Block: {
+    //     number: 0n,
+    //     hash: '0x1218eb233187b0120a09fea9eae5d5360df5abbad66cb59d0f93538fcf4748e5',
+    //     parentHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
+    //     mixHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
+    //     nonce: 0n,
+    //     sha3Uncles: '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347',
+    //     logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+    //     transactionsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+    //     stateRoot: '0xf1613937d7faa51ee731f0f2c7a7828ab0ddd2e4644a268d3be2f1b315b023d1',
+    //     receiptsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+    //     miner: '0x0000000000000000000000000000000000000000',
+    //     difficulty: 0n,
+    //     totalDifficulty: 0n,
+    //     extraData: '0x',
+    //     size: 1000n,
+    //     gasLimit: 6721975n,
+    //     gasUsed: 0n,
+    //     timestamp: 1698721176n,
+    //     uncles: []
+    // }
+}
+(async () => {
+    await test();
+})();
+```
+
+### getBlockNumber
```

Comment:
This is in style of formal documentation , Issue scope is for writing it in tutorial style ( [like](https://docs.web3js.org/guides/smart_contracts/deploying_and_interacting_with_smart_contracts) ). 
Once we will have clean packages documentation this will be overlapping with that (Eth doc). 

I think just add few Eth functions and add link to all remaining eth functions to API, and cover following:
- add only few functions should be enough instead of all ( and may be add reference only as bookmarks at end of tutorial to other eth functions pointing to API ) 
- importing specific package ( mention it will reduce size of user app bundle with actual numbers as compared to importing whole Web3 package)
- example of output formatters ( one example for each types: number, hex, number string, bigint) 
- usage of Eth package level configs ( for some imp variables like txtimeout values, handlerevert, ..etc )